### ISSUE TITLE
VictoryLabel does not support passing rx / ry to the backgroundComponent 

### ISSUE THREAD

**Issue by qjnz** [FormidableLabs/victory#1893]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

The backgroundComponent of [`victory-label`](https://formidable.com/open-source/victory/docs/victory-label) does not seem to support [`rx`/`ry`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/rx) props and I want to be able to make the corner radius for the background. 

https://github.com/FormidableLabs/victory/blob/main/packages/victory-core/src/victory-label/victory-label.js

Is there any way to make the radius work in victory-label?

**Comment by qjnz** [comment]
Hey, @boygirl , thanks for your prompt response. I have tried your suggestion prior to this, it will work if we could include it in the typescript definition since `rx` and `ry` are not included in the `VictoryCommonPrimitiveProps`.
```
export interface VictoryCommonPrimitiveProps {
  active?: boolean;
  ariaLabel?: StringOrCallback;
  className?: string;
  clipPath?: string;
  data?: any;
  desc?: string | Function;
  disableInlineStyles?: boolean;
  events?: object;
  id?: number | string | Function;
  index?: number | string;
  origin?: OriginType;
  polar?: boolean;
  role?: string;
  scale?: any;
  shapeRendering?: string;
  style?: any;
  tabIndex?: NumberOrCallback;
  transform?: string;
}
```
![image](https://user-images.githubusercontent.com/75322/124666900-ca2ff000-df02-11eb-8f3a-6320c4bdb93d.png)

**Comment by boygirl** [comment]
@qjnz Good find! Would you like to open a PR adding optional `rx` and `ry` to that interface?

**Comment by Wesleyzxc** [comment]
Can this be closed? Fixed with v35.10.0 (2021-08-04) https://github.com/FormidableLabs/victory/blob/main/CHANGELOG.md#35100-2021-08-04

**Comment by boygirl** [comment]
@Wesleyzxc correct! thank you!
